sycl/include/CL/sycl/exception.hpp

```diff
@@ -111,5 +117,39 @@ class feature_not_supported : public device_error {
   using device_error::device_error;
 };
 
+enum class errc : unsigned int {
+  runtime = 0,
+  kernel,
+  accessor,
+  nd_range,
+  event,
+  kernel_argument,
+  build,
+  invalid,
+  memory_allocation,
+  platform,
+  profiling,
+  feature_not_supported,
+  kernel_not_supported,
+  backend_mismatch
```

Comment:
Should we add explicit numbers like we agreed to do for every other enum?

Reply:
Will do.